eventmesh-connectors/eventmesh-connector-file/src/main/resources/source-config.yml

```diff
@@ -26,15 +25,7 @@ pubSubConfig:
     passWord: filePassWord
 connectorConfig:
     connectorName: fileSource
-    nameserver: 127.0.0.1:9877
     topic: TopicTest
```

Comment:
`topic` is redundant, too.

Reply:
@pandaapo  Sure, I will update the PR. Just for my knowledge, I wanted to know the usage of **_topic_** in the source connector. Also, in the **_ConnectorConfig_**, the _**topic**_ was used only once, as in every other **_source-config.yml_** file for other source connectors. So, would removing topic be a good idea?

Reply:
`topic` is used as part of the data source definition in other Source Connectors, such as fetching data from a `topic` of Kafka, RocketMQ, or RabbitMQ. The current Connector fetches data from a file, and `filePath` and `fileName` already specify the data source.

Reply:
Updated the PR.Can you please check now?

Reply:
OK. I am now starting the CI for this PR. I would appreciate it if you could test the functionality of this File Source Connector locally. I believe it is a very important job.